### PR TITLE
chapter_11

### PR DESCRIPTION
**Pull request by muizzuddinyafiq**
ready

### CODE REVIEW

**Comment by muizzuddinyafiq** [comment]
ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (ghost) [inline app/assets/stylesheets/account_activations.scss]: delete this file please

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (ghost) [inline app/helpers/account_activations_helper.rb]: delete this file please

File: test/controllers/account_activations_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (ghost) [inline test/controllers/account_activations_controller_test.rb]: delete this file please

File: test/mailers/previews/user_mailer_preview.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_mailer
+class UserMailerPreview < ActionMailer::Preview
```
Comment (ghost) [inline test/mailers/previews/user_mailer_preview.rb]: delete this file please

File: test/mailers/user_mailer_test.rb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+require "test_helper"
+
+class UserMailerTest < ActionMailer::TestCase
+  test "account_activation" do
+    mail = UserMailer.account_activation
+    assert_equal "Account activation", mail.subject
+    assert_equal ["to@example.org"], mail.to
```
Comment (ghost) [inline test/mailers/user_mailer_test.rb]: delete this file please

**Review by ghost — COMMENTED** [review]
cmt

**Comment by muizzuddinyafiq** [comment]
ready

File: config/environments/development.rb [hunk 6]
```diff
@@ -30,23 +40,19 @@
...
   end
 
-  # Store uploaded files on the local file system (see config/storage.yml for options).
+  # Store uploaded files on the local file system (see config/storage.yml for options)
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  host = 'example.com' # Don't use this literally; use your local dev host instead
```
Comment (huongnt-2545) [inline config/environments/development.rb]: remove this line, it's not useful

File: environment variable [hunk 7]
```diff
@@ -0,0 +1,2 @@
+USER_EMAIL="<muizswift1234@gmail.com>"
```
Comment (huongnt-2545) [inline environment variable]: delete this file

**Comment by muizzuddinyafiq** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull request

**Pull request merged**